# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1389113057 18000
#      Tue Jan 07 11:44:17 2014 -0500
# Branch stable
# Node ID faf32a725693e7a73f67d6ebd7f0f2ea5201e887
# Parent  b6b6e0dc700e62d2ddfebc470f8d4ab8105f4cbc
* stmt.txi: Fix missing comma after @xref.

diff --git a/doc/interpreter/stmt.txi b/doc/interpreter/stmt.txi
--- a/doc/interpreter/stmt.txi
+++ b/doc/interpreter/stmt.txi
@@ -841,31 +841,31 @@ end_try_catch
 where @var{body} and @var{cleanup} are both optional and may contain any
 Octave expressions or commands.  The statements in @var{cleanup} are
 only executed if an error occurs in @var{body}.
 
 No warnings or error messages are printed while @var{body} is executing.
 If an error does occur during the execution of @var{body}, @var{cleanup}
 can use the functions @code{lasterr} or @code{lasterror} to access the
 text of the message that would have been printed, as well as its
-identifier. The alternative form,
+identifier.  The alternative form,
 
 @example
 @group
 try
   @var{body}
 catch @var{err}
   @var{cleanup}
 end_try_catch
 @end group
 @end example
 
 @noindent
 will automatically store the output of @code{lasterror} in the structure
-@var{err}. @xref{Errors and Warnings} for more information about the
+@var{err}.  @xref{Errors and Warnings}, for more information about the
 @code{lasterr} and @code{lasterror} functions.
 
 
 @node Continuation Lines
 @section Continuation Lines
 @cindex continuation lines
 @cindex @code{...} continuation marker
 @cindex @code{\} continuation marker
